charts/istioconfig/templates/jwt-checker.yaml

```diff
@@ -0,0 +1,14 @@
+# policy for JWT Verification. All services listed in targets field would require JWT token
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "Policy"
+metadata:
+  name: "jwt-example"
+spec:
+  targets:
+    - name: kernel-service-poc-service
```

Comment:
This list of targets is a good candidate for pulling out into a values. We can have a value that's a list of service names and then loop over them here.
https://helm.sh/docs/chart_template_guide/control_structures/#looping-with-the-range-action